### DISCUSSION TITLE
can't not use prompt for serial-reaction-time-mouse plugin.

### DISCUSSION THREAD

**Discussion by takat0-ch** [jspsych/jsPsych#837] [Q&A] [answered]
Hi,

I am using the serial-reacton-time-mouse plugin. when defining the prompt parameter, can't click grids. Is there any solutions for that?

**Comment by jodeleeuw** [comment, marked as answer]
Hi @takat0-ch 

This is definitely a bug in the plugin.

Try changing line 108 to:

```js
display_element.insertAdjacentHTML('beforeend', trial.prompt);
```

I'll open an Issue to get this patched. Thanks for raising the problem!
